### DISCUSSION TITLE
Sample playback on both channels

### DISCUSSION THREAD

**Discussion by baesek** [marcel-licence/esp32_midi_sampler#73] [Q&A] [answered]
Hi,
I am only getting output on the left speaker when i play back samples on my ESP32 Audio Kit Board V2.2 2957 with ES8388 using the following code:
```
Sampler_NoteOn(2, 40, 1);
Sampler_NoteOff(2, 40);
```
Is this normal if yes, why and how to play back a (mono) sample on both outputs?
- paul

**Comment by marcel-licence** [comment, marked as answer]
Hi,
usually you get sound on both channels (mono).
I've worked on some stereo support but I think I didn't integrate it. That would allow to create a result like this.
You can try to swap channels in the mainloop in the Audio_Output call.
Best regards,
Marcel

**Reply by baesek** [reply]
Thanks for the response!
strange... i just tried to to swap the channels like this:
```
//Audio_Output(fl_sample, fr_sample);
Audio_Output(fr_sample, fl_sample);
```
but i still only hear a sound on the left side.

**Reply by marcel-licence** [reply]
Hm that sounds like a problem with the PCB / electrical connections or maybe with the codec settings. That is the first time I hear about a problem like this.

**Reply by baesek** [reply]
Thank you ! you were right, I just found out, that the headphone jacks right pin is not soldered rightto to the board, when i press it with a pincette I get sound on both speakers.

**Reply by marcel-licence** [reply]
Oh that is crazy. I had also one Audio Kit with to PINs of the ESP32 bridged by too much solder. The quality seems to be a bit poor but at the end it looks like a good value for the low price. I am happy to hear that you found the reason.
